frontend/src/components/ForceScales/ForceScales.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import SliderControl from '../SliderControl/SliderControl.jsx';
+import styles from './ForceScales.css';
+
+/*
+    <SliderControl min='0' max='100' key={'thrust0'} indx={0} val={this.state.scales[0]} inv={this.state.inv[0]} rend={this.rendData.bind(this)} name={"Thruster 0"} />
+*/
+
+export default class ForceScale extends Component {
+    constructor(props) {
+        super(props);
+        this.state = { scales: props.scales };
+
+        this.rendLeftScales = this.rendLeftScales.bind(this);
+        this.rendRightScales = this.rendRightScales.bind(this);
+        this.rendData = this.rendData.bind(this);
+    }
+
+    rendData(val, inv, key) {
+        //console.log(this.state.scales[key]);
+        let scalescpy = this.state.scales;
+        scalescpy[key] = val;
+        //scalescpy[key].invert = inv;
+        this.setState({
+                scales: scalescpy,
+            },
+            function() {
+                this.props.rend(this.state.scales);
+            }
+        );
+    }
+
+    rendLeftScales() {
+        var that = this;
+        return Object.keys(this.props.scales).map(function(val, index) {
+            if (val.startsWith('vel')) {
+                return (
+                    <SliderControl min = "0" max = "100" indx = { val }
+                    power = { that.state.scales[val] } rend = { that.rendData }
+                    name = { 'Force ' + val.slice(-1) } key={'force'+val}
+                    />
+                );
+            }
+        });
+    }
+
+    rendRightScales() {
+        var that = this;
+        return Object.keys(this.props.scales).map(function(val, index) {
+            if (!val.startsWith('vel') && val !== 'master') {
+                return (
+                    <SliderControl min = "0" max = "100" indx = { val }
+                    power = { that.state.scales[val] } rend = { that.rendData }
+                    name = { 'Force ' + val } key={'force'+val}
+                    />
```

Comment:
For code style, I think usually people do this:
```
<SliderControl min="0" max="100" indx={val}
    power = {that.state.scales[val]} rend = {that.rendData}
    name = { 'Force ' + val} key={'force'+val}
/>
```

As in, indent the properties on the other lines. Also, you're a little inconsistent about spaces around variables in `{ }` and spaces around the `=`. Pls pick something :|

Reply:
.....I'm going to blame this on VS Code again.

I tried including a formatting package through a tutorial and it just screwed with everything, even though it was supposed to be for React. So I deleted it all, and it works fine, but my spacings are all screwy now